### PR TITLE
[DROOLS-4611] Fix support for double (infinity and NaN) in test scenario

### PR DESCRIPTION
**Pull request by danielezonca**
This PR fix revert function for double values to support `NaN`, `Infinity` and `-Infinity`

@yesamer @dupliaka @jomarko 

https://issues.redhat.com/browse/DROOLS-4611

### CODE REVIEW

**Review by yesamer — COMMENTED** [review]
To infinite... and beyond!

**Review by yesamer — APPROVED** [review]

File: drools-scenario-simulation/drools-scenario-simulation-backend/src/main/java/org/drools/scenariosimulation/backend/util/ScenarioBeanUtil.java [hunk 1]
```diff
@@ -277,4 +277,11 @@ private static char parseChar(String value) {
     private static String cleanStringForNumberParsing(String rawValue) {
         return rawValue.replaceAll("(-)\\s*([0-9])", "$1$2");
     }
+
+    private static String revertDouble(Double doubleValue) {
+        if(Double.isInfinite(doubleValue) || Double.isNaN(doubleValue)) {
```
Comment (jomarko) [inline drools-scenario-simulation/drools-scenario-simulation-backend/src/main/java/org/drools/scenariosimulation/backend/util/ScenarioBeanUtil.java]: minor formatting `if (Double. .....`

Comment (danielezonca) [reply drools-scenario-simulation/drools-scenario-simulation-backend/src/main/java/org/drools/scenariosimulation/backend/util/ScenarioBeanUtil.java]: Done :)

**Review by jomarko — APPROVED** [review]

**Comment by jomarko** [comment]
thank you @danielezonca

**Pull request merged**